<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>resttime-qt v1.00</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <widget class="QSpinBox" name="sbDecenasHoras">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbUnidadesHoras">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>150</y>
      <width>71</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbDecenasMinutos">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbUnidadesMinutos">
    <property name="geometry">
     <rect>
      <x>543</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>150</y>
      <width>61</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbDecenasSegundos">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbUnidadesSegundos">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>150</pointsize>
     </font>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIniciarCuentaAtras">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>412</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start countdown timer</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPararCuentaAtras">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop countdown timer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMinimizarAlIniciar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimize on start</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbRestaurarAlTerminar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>454</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore on end</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
